integration-tests/tests/analysis-context-bar.js

```diff
@@ -15,16 +15,27 @@ const testAnalysisContextBarFn = _.flow(
   // Navigate to appropriate part of UI (the analysis tab)
   await performAnalysisTabSetup(page, token, testUrl, workspaceName)
 
+  // Ensure UI displays the runtime Terminal icon is present + disabled
+  let iconTooltipText = 'Terminal'
+  await findElement(page, clickable({ textContains: iconTooltipText, isEnabled: false }))
```

Comment:
We try to avoid mutations to make it easier to reason about code (meaning, one wouldn't have to look around the code to see if a variable at hand might be changed somewhere else). In this case, the values are simple string literals, which could be inlined. That'd also save readers of the code having to scroll up/down to see values.

```suggestion
  // Ensure UI displays the runtime Terminal icon is present + disabled
  await findElement(page, clickable({ textContains: 'Terminal', isEnabled: false }))
```

Similarly for below.

If you feel that'd cause too much repetition of the string literals, you could define two `consts`, something like:

```javascript
const tooltipTextTerminal = 'Terminal'
```
```javascript
const tooltipTextEnvCreating = 'Jupyter Environment ( Creating )'
```

Reply:
yeah, I want to avoid text duplication. I'll change to two `const`.